File: javascript/1133_restoDaDivisao.js
// var input = require("fs").readFileSync("dev/stdin/file.txt", "utf8");


// ! Giving Wrong answer 5%

var input = 
`18
10`;

var lines = input.split("\n");

var [x, y] = [parseInt(Math.max(lines[0], lines[1])), parseInt(Math.min(lines[1], lines[0]))];

for (; y < x; y++) {
    if (y % 5 === 2 || y % 5 === 3) {
        console.log(y);
    }
}

File: python3/1133_restoDaDivisao.py
x = int(input())
y = int(input())

if(x > y):
    b = x
    s = y
elif(y > x):
    b = y
    s = x

for s in range(s+1, b):
    if(s % 5 == 2 or s % 5 == 3):
        print(s)
